zsim-ramulator/setup.sh

```diff
@@ -12,5 +12,6 @@ apt-get -y install byacc
 #apt-get -y install libconfig++-dev
 apt-get -y install libhdf5-dev
 apt-get -y install libelf-dev
+apt-get -y install libxerces-c-dev
 
```

Comment:
libboost-all-dev is required as well. Need to add `apt-get -y install libboost-all-dev`